components/V2Layout.tsx

```diff
@@ -33,8 +33,8 @@ const TopNav: React.FC = () => {
         <Navbar.Collapse id="topnav">
           <Nav className="me-auto">
             <NavLink href="/">Home</NavLink>
-            <NavLink href="/browse">Browse</NavLink>
-            {authenticated && <NavLink href="/testify">Testify</NavLink>}
+            <NavLink href="/browse">Bills</NavLink>
```

Comment:
We should consider changing the route itself from `/browse` to `/bills`. To do so we would rename browse.tsx to bills.tsx and replace /browse with /bills wherever it appears.

Also we can go head and delete the Testify line